core/assets/bundles/bundle_vi.properties

```diff
@@ -722,7 +722,7 @@ ability.repairfield = Sửa chữa/Xây dựng
 ability.statusfield = Status Field
 ability.unitspawn = Sản xuất {0}
 ability.shieldregenfield = Tạo khiên nhỏ
-ability.movelightning = Movement Lightning
+ability.movelightning = Bước đi sấm sét
```

Comment:
Lightning step?

Reply:
well , there is no such thing as "Lightning Movement" , and hell is movement too long to use as a practical word , so i chose that